build.gradle

```diff
@@ -64,37 +64,37 @@ subprojects {
         }
     }
 
-//    afterEvaluate {
-//        compileJava {
-//            inputs.property("moduleName", moduleName)
-//            doFirst {
-//                options.compilerArgs = ['--module-path', classpath.asPath,]
-//                classpath = files()
-//            }
-//        }
-//
-//        compileTestJava {
-//            inputs.property("moduleName", moduleName)
-//            doFirst {
-//                options.compilerArgs = ['--module-path', classpath.asPath,
-//                                        '--add-modules', 'junit',
-//                                        '--add-reads', "$moduleName=junit",
-//                                        '--patch-module', "$moduleName=" + files(sourceSets.test.java.srcDirs).asPath,]
-//                classpath = files()
-//            }
-//        }
-//
-//        test {
-//            inputs.property("moduleName", moduleName)
-//            doFirst {
-//                jvmArgs = ['--module-path', classpath.asPath,
-//                           '--add-modules', 'ALL-MODULE-PATH',
-//                           '--add-reads', "$moduleName=junit",
-//                           '--patch-module', "$moduleName=" + files(sourceSets.test.java.outputDir).asPath,]
-//                classpath = files()
-//            }
-//        }
-//    }
+    //    afterEvaluate {
+    //        compileJava {
+    //            inputs.property("moduleName", moduleName)
+    //            doFirst {
+    //                options.compilerArgs = ['--module-path', classpath.asPath,]
+    //                classpath = files()
+    //            }
+    //        }
+    //
+    //        compileTestJava {
+    //            inputs.property("moduleName", moduleName)
+    //            doFirst {
+    //                options.compilerArgs = ['--module-path', classpath.asPath,
+    //                                        '--add-modules', 'junit',
+    //                                        '--add-reads', "$moduleName=junit",
+    //                                        '--patch-module', "$moduleName=" + files(sourceSets.test.java.srcDirs).asPath,]
+    //                classpath = files()
+    //            }
+    //        }
+    //
+    //        test {
+    //            inputs.property("moduleName", moduleName)
+    //            doFirst {
+    //                jvmArgs = ['--module-path', classpath.asPath,
+    //                           '--add-modules', 'ALL-MODULE-PATH',
+    //                           '--add-reads', "$moduleName=junit",
+    //                           '--patch-module', "$moduleName=" + files(sourceSets.test.java.outputDir).asPath,]
+    //                classpath = files()
+    //            }
+    //        }
+    //    }
```

Comment:
Is this intentionally commented out?

Reply:
Yes, I was working on it on the other branch that I'm merging into. It will be uncommented when blackduck-common exposes itself as a module.